doc/reference/actions.rst

```diff
@@ -102,12 +106,27 @@ dialog::
 In-database window actions have a few different fields which should be ignored
 by clients, mostly to use in composing the ``views`` list:
 
-``view_mode``
-    comma-separated list of view types as a string. All of these types will be
+``view_mode`` (default= ``tree, form`` )
+    comma-separated list of view types as a string (/!\ No spaces /!\). All of these types will be
```

Comment:
the two \ has been ignored, the output gives "a string (/!No spaces /!). All of the",